src/common/Widget.js

```diff
@@ -135,6 +135,9 @@
         
         cb.apply(null, objs)
     }
+    if (!window.require) {
```

Comment:
This is to be consistent with how you handled MutableObserver. UMD blocks in the files can just use `root.require` instead of `Widget.prototype.require`.